app/assets/stylesheets/views/_landing_page/card.scss

```diff
@@ -0,0 +1,44 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: govuk-colour("dark-blue");
+  color: govuk-colour("white");
+}
+
+.app-b-card__textbox {
+  padding: govuk-spacing(6) govuk-spacing(7);;
+  background-color: govuk-colour("dark-blue");
+  color: govuk-colour("white");
+
+  .gem-c-govspeak h2 {
```

Comment:
We shouldn't style component elements outside of components - it breaks our principle of isolation. Looks like this only changes the line height - I'd suggest the style coming from the component is acceptable as is.

Reply:
I've removed this whole block now.

Reply:
Thanks!